testkit/src/main/scala/org/bitcoins/testkit/rpc/BitcoindRpcTestUtil.scala

```diff
@@ -391,7 +406,7 @@ trait BitcoindRpcTestUtil extends BitcoinSLogger {
     import system.dispatcher
 
     for {
-      hashes <- clients.head.generate(blocks)
+      hashes <- clients.head.getNewAddress.flatMap(clients.head.generateToAddress(blocks, _))
```

Comment:
rather than using a `flatMap`, you should just make this two lines in the `for` evaluation. I.e.
```
address <- clients.head.getNewAddress()
hashes <- clients.head.generateToAddress(blocks, address)
```